tests/serverTestMultiMappedPV-DoocsVariableConfig.xml

```diff
@@ -3,11 +3,16 @@
 
   <location name="FIRST">
     <property name="SCALAR_1" source="/FLOAT/TO_DEVICE_SCALAR"/>
+    <property name="BIDIRECTIONAL" source="/FLOAT/BIDIRECTIONAL"/>
   </location>
 
   <location name="SECOND">
-    <is_writeable>false</is_writeable>
-    <property name="SCALAR_1" source="/FLOAT/TO_DEVICE_SCALAR"/>
+    <property name="SCALAR_1" source="/FLOAT/TO_DEVICE_SCALAR">
+      <is_writeable>false</is_writeable>
+    </property>
+    <property name="BIDIRECTIONAL" source="/FLOAT/BIDIRECTIONAL">
+      <is_writeable>false</is_writeable>
```

Comment:
Why is writeable "false" for bi-directional. Shouldn't it be "true"?

Reply:
This is a read-only copy of the bi-directional PV - just like we have a read-only copy of the write-only TO_DEVICE_SCALAR. You should not map the same PV twice writeable, as it would create inconsistencies and race conditions.